core/src/main/resources/lib/layout/pane.jelly

```diff
@@ -39,14 +39,17 @@ THE SOFTWARE.
       (so that the title can stretch to the entire table width.
     </st:attribute>
     <st:attribute name="id">
-      @id of the table, if specified.
+      @id of the table, if specified. Required for a pane to be collapsible.
     </st:attribute>
     <st:attribute name="class">
       Pane table class specification.
     </st:attribute>
     <st:attribute name="footer">
       Footer of the box. Can include HTML.
     </st:attribute>
+    <st:attribute name="collapsedText">
```

Comment:
I can't seem to see this being used anywhere

Reply:
line 77